### ISSUE TITLE
Update Ingest with the archive accessions (BioSamples, BioStudies)

### ISSUE THREAD

**Issue by ke4** [ebi-ait/dcp-ingest-central#531]
ENA has separate ticket

**Comment by MightyAx** [comment]
@ke4 testing, will create a draft PR to link here.

**Comment by ke4** [comment]
Draft PR: https://github.com/ebi-ait/ingest-archiver/pull/74

**Comment by jacobwindsor** [comment]
Needs two approvals on the PR but @ke4 is away right now so will be waiting on his return

**Comment by idazucchi** [comment]
Some transitive dependencies caused the archiver to break, not running in staging.

**Comment by Wkt8** [comment]
Hotfix applied, now in production.
